go-template

```diff
@@ -35,23 +35,37 @@ export _GO_STANDALONE=
 declare GO_SCRIPTS_DIR="${GO_SCRIPTS_DIR:-scripts}"
 
 # The `GO_SCRIPT_BASH_REPO_URL` tag or branch you wish to use
-declare GO_SCRIPT_BASH_VERSION="${GO_SCRIPT_BASH_VERSION:-v1.4.0}"
+declare GO_SCRIPT_BASH_VERSION="${GO_SCRIPT_BASH_VERSION:-v1.5.0}"
+
+# The target version string, removing the leading 'v'
+declare _GO_SCRIPT_BASH_VERSION_NUMBER="${GO_SCRIPT_BASH_VERSION:1}"
 
 # The go-script-bash installation directory within your project
 declare GO_SCRIPT_BASH_CORE_DIR="${GO_SCRIPT_BASH_CORE_DIR:-${0%/*}/$GO_SCRIPTS_DIR/go-script-bash}"
 
 # The URL of the go-script-bash framework sources
 declare GO_SCRIPT_BASH_REPO_URL="${GO_SCRIPT_BASH_REPO_URL:-https://github.com/mbland/go-script-bash.git}"
 
+# URL with the release files
+declare GO_SCRIPT_BASH_DOWNLOAD_URL="${GO_SCRIPT_BASH_DOWNLOAD_URL:-${GO_SCRIPT_BASH_REPO_URL%.git}/archive}/$GO_SCRIPT_BASH_VERSION.tar.gz"
+
+
 if [[ ! -e "$GO_SCRIPT_BASH_CORE_DIR/go-core.bash" ]]; then
-  printf "Cloning framework from '%s'...\n" "$GO_SCRIPT_BASH_REPO_URL"
-  if ! git clone --depth 1 -c advice.detachedHead=false \
-      -b "$GO_SCRIPT_BASH_VERSION" "$GO_SCRIPT_BASH_REPO_URL" \
-      "$GO_SCRIPT_BASH_CORE_DIR"; then
-    printf "Failed to clone '%s'; aborting.\n" "$GO_SCRIPT_BASH_REPO_URL" >&2
+  printf "Downloading framework from '%s'...\n" "$GO_SCRIPT_BASH_DOWNLOAD_URL"
+  curl -LfsS "$GO_SCRIPT_BASH_DOWNLOAD_URL" | tar -xz
+  if [[ ${PIPESTATUS[0]} -ne 0 ]] || [[ ${PIPESTATUS[1]:1} -ne 0 ]] ; then
```

Comment:
Hmm, you may be right about `pipefail`. And what I've normally done to avoid pipes throughout the framework is to use process substitutions, so maybe this would be worth a try:

```bash
if command -v curl >/dev/null && command -v tar >/dev/null &&
   tar -xz <(curl -LfsS "$GO_SCRIPT_BASH_DOWNLOAD_URL); then
```

As for making `command -v` fail, you can prefix the `run` command like so:

```bash
PATH="$BATS_TEST_BINDIR" run ...
```
And check out `stub_program_in_path` and `restore_program_in_path` from `lib/bats/helpers` to stub out `curl` and `tar`.